style.css

```diff
@@ -37,81 +46,138 @@
     --color-gdscript: var(--nord-frost-3);
 
     --icon-padding: 0.5em;
+
+    --spacer: 4em;
+
+    --font-size-sm: .8rem;
+    --font-size-med: 1rem;
+    --font-size-lg: 1.2rem;
+    --font-size-xl: 1.5rem;
 }
 
 /* General */
-.no-style {
-    list-style: none;
-    padding: 0;
+html {
+    background: var(--color-bg);
 }
+
 body {
-    background: var(--color-bg);
     color: var(--color-fg);
     line-height: 1.6;
-    max-width: 60vw;
-    padding-left: 20vw;
-    padding-right: 20vw;
+    max-width: 90vw;
+    padding-left: 5vw;
+    padding-right: 5vw;
+    margin-top: calc(var(--spacer) / 2);
+    margin-bottom: var(--spacer);
     font-family: monospace;
-    font-size: 16pt;
+    font-size: var(--font-size-sm);
 }
-h1,h2,h3,h4 {
+
+h1,
+h2,
+h3,
+h4 {
     line-height: 1.2;
 }
+
 a {
     text-underline-offset: 0.3em;
+
+    &:link {
+        color: var(--color-link);
+    }
+
+    &:hover,
+    &:focus-visible {
+        color: var(--color-link-hover);
+    }
+
+    &:visited:not(.navbar li a):not(.project-header a) {
+        color: var(--color-link-visited);
+    }
+
+    &:has(code) {
+        text-decoration: none;
+    }
 }
-a:link {
-    color: var(--color-link);
-}
-a:visited {
-    color: var(--color-link-visited);
-}
-a:hover {
-    color: var(--color-link-hover);
+
+code {
```

Comment:
Because you are using monospace for everything, `code` blocks were getting lost. I added in some styling to make them stand out. if you don't like it def change, but I would do something to make them appear :)